.goreleaser.yml

```diff
@@ -54,6 +54,8 @@ dockers:
       - "bbvalabsci/kapow:{{ .Tag }}"
       - "bbvalabsci/kapow:v{{ .Major }}"
 
+    skip_push: true
+
 release:
   draft: false
```

Comment:
We need to skip the publishing of docker images, no because rc's, but when Docker credentials are not present (for forks to be able to run the CI without problems)

Reply:
Oh, I see. You are pushing the docker images by hand. Got it.